src/commissaire_service/transport/ansible_wrapper.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (C) 2016  Red Hat, Inc
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# This is Steve Milner's programmatic Ansible hack (the good kind) from:
+# https://stevemilner.org/2016/07/30/programmatic-ansible-middle-ground/
+
+import json
+import logging
+import os  # Used for expanding paths
```

Comment:
nit: Looks like ``os.path`` is the only part of ``os`` in use. Maybe ``import os.path`` instead.